frontend/src/components/common/TermsAndConditions.vue

```diff
@@ -6,6 +6,19 @@ import {
     hideTerms,
     isTermsCloseable,
 } from '@/store/TermsAndConditionsState';
+import { AppActlApiService } from '@/services/ApiServiceFactory';
+import { setRouteToastError } from '@/store/ToastState';
+
+const acceptTermsAndConditions = async () => {
+    try {
+        await AppActlApiService.userTermsAndConditionsApi.createUserTermsAndConditions();
+        hideTerms();
+    } catch (error: any) {
+        // temporary error handling
+        setRouteToastError(error.message);
```

Comment:
Looks correct to me but just wondering is the toast error able to show up right after 'logout', have you tried?

Reply:
You mean show up in the landing page?

Reply:
Then please don't put the code there for others wondering if what you intended to do isn't working (if you already verify and know it isn't working) due to some reason. 
For now, please remove the toast error, only just logout.
Thank you.

Reply:
It's working, if there is an error while accepting it will show a  toast, but the toast is not on the landing page

Reply:
I see, the transition might be too fast that user does not even have time to know what error it is.
Sure, then leave it, it is a limitation of current toast handling.
Thanks.